test/application/onerror.js

```diff
@@ -48,7 +48,7 @@ describe('app.onerror(err)', () => {
 
     const output = stderr.inspectSync(() => app.onerror(err));
 
-    output.should.eql(['\n', '  Foo\n', '\n']);
+    output.should.eql(['\n  Foo\n\n']);
```

Comment:
It's only due to the (much less) clean change above. However, my comment on it may be a good compromise.